crates/ra_db/src/input.rs

```diff
@@ -108,11 +108,45 @@ struct CrateData {
     file_id: FileId,
     edition: Edition,
     dependencies: Vec<Dependency>,
+    cfg_options: CfgOptions,
+}
+
+#[derive(Debug, Clone, PartialEq, Eq, Default)]
+pub struct CfgOptions {
+    test: bool,
```

Comment:
I don't see where `test` is used.

Reply:
Oh, it's removed. Seems derives suppress "field never used" warning.